components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -567,10 +600,23 @@ - (void)layoutSubviews {
   }
 }
 
-- (BOOL)isJustifiedLayoutStyle {
-  CGSize contentSize = [self availableSizeForSubviewLayout];
-  CGFloat requiredWidth = [self intrinsicContentSizeForJustifiedLayout].width;
-  return contentSize.width >= requiredWidth;
+- (BOOL)isScrollableLayoutStyle {
+  return [self layoutStyle] == MDCTabBarViewInternalLayoutStyleScrollable;
+}
+
+- (MDCTabBarViewInternalLayoutStyle)layoutStyle {
+  CGSize availableSize = [self availableSizeForSubviewLayout];
+  CGFloat requiredWidthForJustifiedLayout = [self intrinsicContentSizeForJustifiedLayout].width;
+  if (availableSize.width < requiredWidthForJustifiedLayout) {
+    return MDCTabBarViewInternalLayoutStyleScrollable;
+  }
+  if (self.items.count == 0) {
+    return MDCTabBarViewInternalLayoutStyleFixedJustified;
```

Comment:
Should this be the first thing to check in this method to remove any calculations and just return if there are no items.

Reply:
Done.